### PR TITLE
Add Slack Documentation

### LINKED ISSUE #392: Add documentation for Slack
Should be a point of reference for prospective staff. We can link to this from the first meeting presentation.

### PR DESCRIPTION
**Pull request by kkuehler**
This is in preperation for the first general meeting of Fall 2018.

Resolves #392

### CODE REVIEW

File: ocfweb/docs/docs/contact/slack.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+[join]: https://ocf.io/join
+[slack]: https://fco.slack.com
+
+### Optional: Using wee-slack
+
+Note: This section is targeted at IRC users who would like to access Slack
+using the `weechat` IRC client. If that sentence is meaningless to you, save
+yourself some time, stop reading, and access Slack via the desktop and mobile
+apps like a normal person.
```
Comment (dkess) [inline ocfweb/docs/docs/contact/slack.md]: I'd rephrase this.

Comment (abizer) [reply ocfweb/docs/docs/contact/slack.md]: 3edgy5me
